code/game/objects/structures/fireaxe.dm

```diff
@@ -130,7 +130,7 @@
 		locked = !locked
 		update_icon()
 
-/obj/structure/fireaxecabinet/verb/toggle_open() //nice name, huh? HUH?! -Erro //YEAH -Agouri
```

Comment:
Damn humans being exterminated by the oranges we need a doctor to stop this :laughing: